### PR TITLE
Changed font, added rudimentary clock, etc

### PR DESCRIPTION
**Pull request by mathcolo**
There's a clock now:

image

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -11,6 +11,6 @@ app.addRoute('/404', require('./pages/404'))
 
 app.addRoute('/privacy', require('./pages/privacy'))
 app.addRoute('/opensource', require('./pages/opensource'))
-app.addActions({ update: require('./actions/update')})
+app.addActions({ update: require('./actions/update'), clock: require('./actions/clock')})
```
Comment (JRJurman) [inline main.js]: is this file included?

File: pages/home.js [hunk 2]
```diff
@@ -17,21 +18,26 @@ const download = (store, actions) => {
...
     
   }
 };
 
+
 module.exports = (store, actions) => {
-  download(store, actions);
+  window.setInterval(() => {
+    download(store, actions);
+  }, 5000);
```
Comment (JRJurman) [inline pages/home.js]: I would potentially have this logic in an action that triggers the fetch.
You need some kind of check here to make sure you only do this once (which is why having a dedicated reducer is useful).

Comment (JRJurman) [reply pages/home.js]: see: https://github.com/JRJurman/binary-clock/tree/master/actions

File: public/index.html [hunk 3]
```diff
@@ -7,6 +7,8 @@
   <title>Boston Subway Vehicles</title>
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.2/css/bulma.min.css">
   <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+  <link rel="stylesheet" href="/styles.css">
```
Comment (JRJurman) [inline public/index.html]: does styles.css exist?

File: elements/line-status-line.js [hunk 4]
```diff
@@ -6,32 +6,38 @@ const nl_status = (status) => {
...
+    const trains = store.update[attrs.line];
+
     if(attrs.line == 'Blue') {
         return html`<div>No new trains expected on the Blue Line.</div>`
     }
 
+    const nothingNew = html`<div>No new trains on this line, yet.</div>`;
+
+    const trainsNew = trains.filter((train) => train.cars_new_flag);
+    const trainsOld = trains.filter((train) => !train.cars_new_flag);
```
Comment (JRJurman) [inline elements/line-status-line.js]: this is nice 👍

File: main.js (hunk 1 above)
Comment (mathcolo) [reply main.js]: Whoops :( I'll add these; I forgot!

File: public/index.html (hunk 3 above)
Comment (mathcolo) [reply public/index.html]: Yup, it's actually in this PR! :-)

**Pull request merged**